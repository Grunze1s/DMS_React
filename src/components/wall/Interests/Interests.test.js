import React from 'react';
import Interests from './index';
import {
    configure,
    shallow
} from 'enzyme';
import EnzymeAdapter from 'enzyme-adapter-react-16';
// JestHook.mock('expo-font');
configure({
    adapter: new EnzymeAdapter
})

const dummy_data={
    interestList:[{id:"1", interest:"interest1"},
    {id:"2", image:"interest2"}]
}
const setup = (props = {}, state = null) => {
    return (shallow( < Interests {...props}  />)
    )
}

describe('<Interests />', () => {
  

it("Interests render", () => { 
        const wrapper= setup({...dummy_data});
        expect(wrapper).toMatchSnapshot();
  
})
})

