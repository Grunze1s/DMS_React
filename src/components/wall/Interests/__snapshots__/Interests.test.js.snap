// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Interests /> Interests render 1`] = `ShallowWrapper {}`;
